sig-docs/annual-report-2021.md

```diff
@@ -4,79 +4,79 @@
+   - Localization Subproject: SIG Docs is working on formalizing the localization work that has been ongoing for some time, with appointed leads of this initiative as well as recognizing the contributions of various community members across the different languages the Kubernetes website has been translated into. This subproject will be finalized by Q1 2022, with all active localizations informed and updated. The issue tracking the formalizing process can be viewed [here](https://github.com/kubernetes/website/issues/31955).
+   - New Contributor Ambassador Program: As a continuation of our push to grow the SIG Docs contributor base, we're working on a specalized role that aims to support new and would-be contributors get up to speed with our processes and workflows. This role would be capped at six months for it to be shared amongst the community, with this feeding into a possible reviewer funnel as contributors get more comfortable with providing feedback to others. The issue tracking the formalization and documentation of this role can be viewed [here](https://github.com/kubernetes/website/issues/31946).
 
-3. KEP work in 2021 (1.x, 1.y, 1.z):
+3. KEP work in 2021 (continuous and does not target any release):
 
 <!--
 In future, this will be generated from kubernetes/enhancements kep.yaml files
 1. with SIG as owning-sig or in participating-sigs
 2. listing 1.x, 1.y, or 1.z in milestones or in latest-milestone
 -->
 
-   - Stable
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.stable
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.stable
-   - Beta
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
-   - Alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-   - Pre-alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md)
+   - [1326 - Doc policies for third party content](https://git.k8s.io/enhancements/keps/sig-docs/1326-third-party-content-in-docs/README.md)
+
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
+   - Require more contributors to form a stable pool for the blog subproject, in particular and SIG Docs, generally.
+   - Require guidelines on how we could ramp up folks nominated to the New Contributor Ambassador role specific to SIG Docs. 
 
```

Comment:
I hope this is written from a sufficiently neutral point of view
```suggestion
    - One localization (Ukrainian) is primarily worked on by people based in Ukraine,
      where the ongoing and intensifying conflict creates challenges that take priority over
      open source contribution.
```

I used “conflict” rather than “invasion” because I imagine that the CNCF would rather take a neutral stance on the Russian invasion.